client_lib/pulp/client/commands/repo/sync_publish.py

```diff
@@ -190,13 +193,14 @@ def __init__(self, context, renderer, distributor_id, name='run', description=DE
                                         options will override respective options from the default
                                         publish config. Each entry should be either a PulpCliOption
                                         or PulpCliFlag instance
-        :type override_config_options: list
+        :type override_config_options: tuple
         """
         super(RunPublishRepositoryCommand, self).__init__(name, description, method, context,
                                                           renderer)
 
         self.distributor_id = distributor_id
         self.override_config_keywords = []
+        override_config_options += (FLAG_FORCE_FULL, )
```

Comment:
Will this make the flag appear for every plugin that uses this base class? Since only ``pulp_rpm`` is going to support it (for now at least), it would be better to only have it show up for that plugin.